core/src/js/services/mercenaries/highlights/mercenaries-synergies-highlight-processor.ts

```diff
@@ -619,7 +652,12 @@ const or = (...selectors: HighlightSelector[]): HighlightSelector => {
 	return (card: ReferenceCard) => selectors.some((selector) => selector(card));
 };
 
+const not = (selector: HighlightSelector): HighlightSelector => {
+	return (card: ReferenceCard) => !selector(card);
+};
+
 const merc = (card: ReferenceCard) => card.mercenary;
+const minion = (card: ReferenceCard) => !!card.race && !card.mercenary;
```

Comment:
What happens if the minion doesn't have any tribe?

Reply:
I also thought that this might not be the best approach. But we are not highlighting creatures themselves, but abilities that summon them, and it wouldn't be correct to define creatures through cardtype. I also tried to define it using the LETTUCE_ABILITY_SUMMONED_MINION tag, but for some reason it doesn't work. In practice, all summoned creatures have a tribe. If you tell me how to implement it better, then I will do it)

Reply:
The tag is not translated into anything in the cards.json file, but if necessary I can add a flag.
However from what you're saying I think that the approach you have right now is good enough, and we'll adjust if need be in the future.